framework/web/UrlRule.php

```diff
@@ -96,6 +122,11 @@ class UrlRule extends Object implements UrlRuleInterface
      * @since 2.0.10
      */
     public $normalizer;
+    /**
+     * @var int|null status of the URL creation after the last [[createUrl()]] call.
+     * @since 2.0.12
+     */
+    public $createStatus;
```

Comment:
The following discussion was not resolved https://github.com/yiisoft/yii2/pull/13350#discussion_r114478945

I think this property should not bt public.